frontend/src/bundles/talent-onboarding/components/candidate-profile/profile-first-section/styles.module.scss

```diff
@@ -0,0 +1,173 @@
+ul {
```

Comment:
B10 - https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

Reply:
I removed it and the default list-style-type doesn`t appear.